# authentication error messages
authentication.err.validation.authenticator.notfound=Autenticazione non riuscita. Autenticatore con il nome {0} non trovato.
authentication.err.validation.authenticator.notactive=L'autenticatore non \u00e8 attivo

authentication.err.authentication=Autenticazione non riuscita. Nome utente o password non corretti. Nome utente:{0} Motivo {1}
authentication.err.connection=Impossibile effettuare la connessione a {0}. Motivo {1}
authentication.err.communication=Impossibile comunicare con {0}. Motivo {1}

# LDAP error messages
authentication.err.connection.ldap.authenticator.unknownhost=Impossibile effettuare la connessione. Host ldap {0} sconosciuto
authentication.err.connection.ldap.user.notfound=Utente LDAP {0} non trovato
authentication.err.connection.ldap.manager.notfound=Utente Gestione LDAP {0} non trovato
authentication.err.connection.ldap.search=Impossibile cercare LDAP. Motivo {0}

# PASSTHRU
authentication.err.connection.passthru.server=Impossibile aprire una sessione con il server passthru
authentication.err.passthru.token.unsupported=Tipo di token di autenticazione non supportato
authentication.err.passthru.guest.notenabled=Accessi guest disabilitati
authentication.err.passthru.user.disabled=Account disabilitato
authentication.err.passthru.user.notfound=Utente Passthru {0} non trovato

# Authentication Diagnostic Steps
authentication.step.ldap.validation=Convalida della richiesta
authentication.step.ldap.connecting=Connessione al server LDAP {0} in corso 
authentication.step.ldap.connected=Connesso al server LDAP {0} con il server principale: {1}
authentication.step.ldap.lookup=Ricercare l''userId:{0} dell''utente test con la query: {1}
authentication.step.ldap.lookedup=UserId:{0} dell''utente test cercato. Nome distinto (DN) trovato:{1}
authentication.step.ldap.format.user=Formato del nome utente specificato. UserId :{0} dell''utente con il formato userNameFormat:{2} per creare un nome distino (DN):{1}
authentication.step.ldap.authentication=Autentica userId:{0}


# Sync Summary message 
synchronization.summary.status={0} utenti e {1} gruppi elaborati


# Error messages
authentication.ldap.validation.authenticator.notfound=Autenticatore non trovato

# Synchronization Diagnostic Steps
synchronization.step.users={0} utenti trovati
synchronization.step.groups={0} gruppi trovati
synchronization.step.users.lastsynced=Ultima sincronizzazione degli utenti riuscita {0}
synchronization.step.groups.lastsynced=Ultima sincronizzazione dei gruppi riuscita {0}

synchronization.err.ldap.search=Errore durante la ricerca LDAP. Motivo:{0}
synchronization.err.ldap.get.user.id.missing=DN con attributo ID utente mancante = {0}, attributeName = {1})
synchronization.err.ldap.get.group.id.missing=DN con attributo ID gruppo mancante = {0}, attributeName = {1})

synchronization.err.ldap.search.base.invalid=Base di ricerca non valida {0} motivo: {1}

synchronization.err.ldap.group.member.missing.exception="Impossibile risolvere il membro del gruppo {0} con un nome distinto: {1}, motivo {2});
synchronization.err.ldap.group.member.missing="Impossibile risolvere il membro del gruppo {0} con un nome distinto: {1});



       



